app/views/contexts/section_summary_context.py

```diff
@@ -91,6 +105,18 @@ def _custom_summary_elements(self, section_summary, current_location, section):
                     summary_element, current_location, section
                 )
 
+    def _resolve_custom_page_title(
+        self, page_title: str, current_location: Location
+    ) -> str:
+        if list_item_id := current_location.list_item_id:
+            if for_list := current_location.list_name:
```

Comment:
Can these be made a single if?

Reply:
Yes - in the next push this will be changed in any case